src/main/java/io/github/syst3ms/skriptparser/sections/SecLoop.java

```diff
@@ -74,15 +74,13 @@ public boolean init(Expression<?>[] expressions, int matchedPattern, ParseContex
 	public Optional<? extends Statement> walk(TriggerContext ctx) {
 		if (isNumericLoop) {
 			var range = (BigInteger[]) times.getSingle(ctx)
```

Comment:
It's an array of BigInteger, so why use ``var range`` instead of ``BigInteger[] range``?

Reply:
The `var` keyword is like a shortcut. Instead of having to write the type you can just use this keyword to take in the returned type defined at compile-time (I think). I do not think this is necessary, but I’ll look after it and see if it’d be more readable with an explicit type declaration.